app/templates/public.hbs

```diff
@@ -14,7 +14,7 @@
               <h5 class="event time ends">{{t 'To'}} {{header-date this.model.endsAt this.model.timezone}}</h5>
             {{/if}}
             <h1 class="event name">{{this.model.name}}</h1>
-            <h4 class="event location"><i class="icon map marker alternate"></i>{{this.model.locationName}}</h4>
+            <h4 class="event location"><i class="icon map marker alternate"></i>{{this.model.headerLocation}}</h4>
```

Comment:
```suggestion
            <h4 class="event location"><i class="icon map marker alternate"></i>{{this.headerLocation}}</h4>
```

Reply:
Do not resolve conversation. It is resolved automatically once suggested changes are made